src/main/java/core/basesyntax/service/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Lottery {
+    ColorSupplier colorSupplier = new ColorSupplier();
+    NumberSupplier numberSupplier = new NumberSupplier();
+
+    public Ball getRandomBall() {
+        Ball ball = new Ball();
+        ball.setColor(colorSupplier.getRandomColor());
+        ball.setNumber(numberSupplier.getRandomNumber());
```

Comment:
There are no method getRandomNumber in NumberSupplier class, please implement it, also pay attention that ball number maximum value is restricted